README.md

```diff
@@ -100,6 +102,29 @@ shouldRetry([Error || FetchResponse]) returns bool
 
 You can use any attribute of the [FetchResponse](https://developer.mozilla.org/en-US/docs/Web/API/Response) or Error to determine whether to retry or not. Your function _must_ handle both errors (such as network errors) and FetchResponse objects without blowing up. We recommend stateless, side-effect free functions. You do not need to worry about the maximum number of retries -- promise-retry will stop retrying after the maximum you specify. See the tests and `src/retriers.js` file for examples.
 
+### On Request callbacks
+
+We've implemented two callbacks functions that can be passed as part of the **jsonFetch** options to do something `onRequestStart` and `onRequestEnd`, this is specially helpful to log request and response data in retry calls.
+
+All `jsonFetch` options are availables parameters for this two callback functions plus
+
+```typescript
+url?: string;
+retryCount?: number;
+responseOrError?: Response | Error | unknown;
+```
```

Comment:
Clarifying a bit:
```suggestion
Two callback functions can be passed as options to do something `onRequestStart` and `onRequestEnd`. This may be especially helpful to log request and response data on each request.

If you have retries enabled, these will trigger before and after each _actual, individual request_.

#### `onRequestStart`

If given, `onRequestStart` is called with:
```typescript
{
  // ... all the original json-fetch options, plus:
  url: string;
  retryCount: number;
}
```

#### `onRequestEnd`

If given, `onRequestEnd` is called with:
```typescript
{
  // ... all the original json-fetch options, plus:
  ur?: string;
  retryCount: number;
  responseOrError: Response | Error;
}
```
```

Reply:
Ugh, GitHub can't handle the syntax of Markdown suggestions because it's Markdown within Markdown... Hopefully you can figure out what I meant!

Reply:
LOL